main/src/ca/uwaterloo/flix/api/lsp/provider/CompletionProvider.scala

```diff
@@ -481,6 +483,87 @@ object CompletionProvider {
     }.toList
   }
 
+  /**
+    * Format type params in the right form to be inserted as a snippet
+    */
+  private def formatTParamsSnippet(tparams: List[TypedAst.TypeParam]): String = {
+    tparams match {
+      case Nil => ""
+      case _ => tparams.zipWithIndex.map {
+                  case (tparam, idx) => "$" + s"{${idx + 1}:${tparam.name}}"
+                }.mkString("[", ", ", "]")
+    }
+  }
+
+  /**
+    * Format type params in the right form to be displayed in the list of completions
+    */
+  private def formatTParams(tparams: List[TypedAst.TypeParam]): String = {
+    tparams match {
+      case Nil => ""
+      case _ => tparams.map(_.name).mkString("[", ", ", "]")
+    }
+  }
+
+  val simpleTypeNames = List(
```

Comment:
Lets move this to the top of the file and call it `BuiltinTypes` or `BuiltinTypeNames` 

Also add a comment like:

// This list manually maintained. If a new built-in type is added, it must be extended.
// Built-in types are typically descrbed in `TypeConstructor`, `Namer` and `Resolver`.

(SimpleType has a very specific meaning elsewhere in the compiler.)

Reply:
That makes sense. Should I also add a comment in one or more of TypeConstructor etc. pointing back here?

Reply:
You could, yes!